ktor-server/ktor-server-test-host/jvm/src/io/ktor/server/testing/client/TestHttpClient.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2014-2019 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.testing.client
+
+import io.ktor.client.engine.*
+import io.ktor.server.testing.*
+
+/**
+ * A [TestHttpClientConfig] connected to the application engine.
+ */
+val TestApplicationEngine.TestHttpClient: HttpClientEngineFactory<TestHttpClientConfig>
```

Comment:
@InternalAPI

Reply:
No, this is not internal, this is a shortcut

Reply:
So one could write

```kotlin
HttpClient(TestHttpClient).use { client ->   ... }
```